src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { ReactElement, useState } from "react";
+import { PopoverProps } from "./PopoverTypes";
+import {
+	ContentWrapper,
+	StyledPaper,
+	PopperSx,
+	Arrow,
+	StyledHr,
+} from "./Popover.styled";
+
+// Components
+import Popper from "@mui/material/Popper";
+import Grow from "@mui/material/Grow";
+import ClickAwayListener from "@mui/material/ClickAwayListener";
+import Paper from "@mui/material/Paper";
+
+const Popover = (props: PopoverProps): ReactElement => {
+	const {
```

Comment:
I recommend throwing an error if no anchorEl was passed saying something like "Popover component requires an anchor element to position itself but none was provided", and also another error if top and bottom content are not passed saying something like "Popover component expected some content but none was provided".